app/src/routes/api/v2/area.router.js

```diff
@@ -364,22 +400,31 @@ class AreaRouterV2 {
         ctx.statusCode = 204;
     }
 
-    static async getAlertsOfArea(ctx) {
-        logger.info(`Obtaining alerts of area with id ${ctx.params.id}`);
-        ctx.assert(ctx.query.precissionPoints, 400, 'precissionPoints is required');
-        ctx.assert(ctx.query.precissionBBOX, 400, 'precissionBBOX is required');
-        const result = await AreaModel.findOne({ _id: ctx.params.id });
-        if (!result) {
-            ctx.throw(404, 'Area not found');
+    static async updateByGeostore(ctx) {
```

Comment:
this should probably be a middleware

Reply:
Moved ADMIN validation to middleware :+1: